### PR TITLE
feat(dashboard): gh #602 Added cluster banner + updated look and feel for delete button

### PR DESCRIPTION
**Pull request by abujeda**
Added cluster banner to the home page when no sessions are running.
Updated delete session button to make it less prominent.

Part of the UI improvements after the first deployment to the FASSE cluster.

### CODE REVIEW

**Review by whorka — CHANGES_REQUESTED** [review]
This implementation hard-codes configuration values which are site-specific. Please retool the logic to load the `sid_welcome_message` HTML from a configuration file instead (to be delivered via the `docker compose` step in the `Makefile` in the Local Development and Remote Development environments; and to be delivered by Puppet in the Staging and Production environments).

In general, we should avoid hard-coding site-specific values, since this project is intended to be open source, and will be publicized to the wider OOD user community on Thursday.

File: dashboard/application/app/assets/stylesheets/quick_launch.scss [hunk 1]
```diff
@@ -7,6 +7,10 @@ $launcher-border-color: #337ab7;
 $launcher-border-color-secondary: #808080;
 $launcher-color-brown: #DA6027;
 
+$development-color: #255A8E;
+$cannon-color: #009CD2;
+$fasse-color: #DA6027;
```
Comment (whorka) [inline dashboard/application/app/assets/stylesheets/quick_launch.scss]: site-specific

**Comment by abujeda** [comment]
@whorka started working on a different approach.
Making the cluster banner a widget and including the widget based on the current cluster id.

I still need to make more changes to complete this approach, but I would like to discuss it with @whorka

**Comment by abujeda** [comment]
I created an improved approach based on the custom configuration per cluster changes from issue https://github.com/hmdc/DevOpsProjects/issues/598

This approach extends the quick links configuration file created for 598 to configure as well the welcome message.
Edit: Forced update from the improved branch as it is a better solution. To discuss at the daily.

**Comment by abujeda** [comment]
Closing, changes added to #35

**Pull request closed without merging**